databaseConnection.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost/testDb',{
```

Comment:
Let's make this a function. Export a function that connects to the db when called.

Reply:
Changes made